client/plots/brainMRI.js

```diff
@@ -0,0 +1,156 @@
+import { getCompInit, copyMerge } from '#rx'
+import { multiInit } from '../rx'
+import { topBarInit } from './controls.btns'
+import { configUiInit } from './controls.config'
+import { dofetch3 } from '#common/dofetch'
+
+class BrainMRI {
+	constructor(opts) {
+		this.opts = opts
+		this.type = 'brainMRI'
+		this.isOpen = true
+	}
+
+	async init() {
+		const holder = this.opts.holder
+		const controlsHolder = holder.append('div').style('display', 'inline-block').style('vertical-align', 'top')
+		const topbar = controlsHolder.append('div')
+		const config_div = controlsHolder.append('div')
+		const configInputsOptions = this.getConfigInputsOptions()
+
+		this.features = await multiInit({
+			topbar: topBarInit({
+				app: this.app,
+				id: this.id,
+				downloadHandler: () => {
+					const imgElement = holder.select('div[id="sjpp_brainMRI_holder_div"] img').node()
+					const downloadImgName = 'brainMRI'
+					const a = document.createElement('a')
+					document.body.appendChild(a)
+
+					a.addEventListener(
+						'click',
+						() => {
+							// Download the image
+							a.download = downloadImgName + '.png'
+							a.href = imgElement.src
+							document.body.removeChild(a)
+						},
+						false
+					)
+					a.click()
+				},
+				callback: () => this.toggleVisibility(this.isOpen),
+				isOpen: () => this.isOpen,
+				holder: topbar
+			}),
+
+			config: configUiInit({
+				app: this.app,
+				id: this.id,
+				holder: config_div,
+				isOpen: () => this.isOpen,
+				inputs: configInputsOptions
+			})
+		})
+	}
+
+	getConfigInputsOptions() {
+		const configInputsOptions = []
+		const mandatoryConfigInputOptions = [
+			{
+				label: 'Sagittal',
```

Comment:
later may declare these brain-specific options in ds, thus allow the app to be generic and applicable to other organ